app/Http/Controllers/Admin/UserController.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+namespace App\Http\Controllers\Admin;
+
+use App\Models\User;
+use App\Http\Resources\UserResource;
+use App\Http\Requests\User\UpdateUserRequest;
+use App\Http\Requests\User\StoreUserRequest;
+use Symfony\Component\HttpFoundation\Response;
+use App\Http\Controllers\Controller;
+
+class UserController extends Controller
+{
+
+    public function index()
+    {
+        $users = User::all();
+        return UserResource::collection($users);
+    }
+
+    public function show(User $user)
+    {
+        return new UserResource($user);
```

Comment:
what if it fails ? error handling in case the user id does not exist must be added